src/libslic3r/GCodeWriter.cpp

```diff
@@ -249,6 +257,9 @@ std::string GCodeWriter::set_jerk_xy(double jerk)
 
 std::string GCodeWriter::set_accel_and_jerk(unsigned int acceleration, double jerk)
 {
+    double jerk_x = jerk;
```

Comment:
jerk_x/y is not supported by klipper, and these two valriables are not used here.
We should revert changes in this function

Reply:
Looks reasonable?
```
    // Clamp the jerk to the allowed maximum.
    if (m_max_jerk_x > 0 && jerk > m_max_jerk_x)
        jerk = m_max_jerk_x;
    if (m_max_jerk_y > 0 && jerk > m_max_jerk_y)
        jerk = m_max_jerk_y;
```

Reply:
yes

Reply:
Can we remove this two unused lines?

Reply:
They are used, we still need to cap single jerk to slowest axis.

Reply:
I updated PR branch.

Reply:
I see it's still here.
These two variables are not used in this function, so they can be deleted.

Reply:
Sorry, I didn't get you are talking about variable declarations in `set_accel_and_jerk`. Removed.